Rasa_Bot/domain/domain_goal_setting_dialogs.yml

```
@@ -1,6 +1,19 @@
 version: "3.1"
 
+intents:
+  - goal_setting_dialog
+  - EXTERNAL_goal_setting_dialog

```

```suggestion
  - EXTERNAL_trigger_goal_setting

```
This is the definition in the nlu file. 
Also, we can get rid of the testing intent